src/ui/molecules/accordion/index.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import styled, { StyledComponent } from 'styled-components';
+import { ArrowDown } from 'icons';
+import { Variant } from 'lib/types';
+
+interface AccordionProps {
+  className?: string;
+  isOpen: boolean;
+  title?: React.ReactNode;
+}
+
+const AccordionBase: React.FC<AccordionProps & Variant> = ({
+  children,
+  className,
+  isOpen = true,
```

Comment:
```suggestion
  isOpened = true,
```

`open` это глагол, он не может описывать состояние. В русском аналогично, нельзя использовать "открыть" для описания состояния, мы используем специальную форму "открыто", в английском аналогично `is opened`